helper_scripts/cloudstack/build_run_deploy_test.sh

```diff
@@ -417,9 +421,18 @@ date
 # Run the tests
 if [ ${run_tests} -eq 1 ]; then
   echo "Running Marvin tests.."
-  bash -x /data/shared/helper_scripts/cloudstack/run_marvin_router_tests.sh ${marvinCfg}
+  if [ -z ${test_file} ]; then
+    # for backwards compatibility, default to 'run_marvin_router_tests.sh' if test file is not specified
+    test_file="run_marvin_router_tests.sh"
+  fi
+  if [ -f "/data/shared/helper_scripts/cloudstack/${test_file}" ]; then
```

Comment:
I can't seem to edit on my phone apparently. :) 

I will check on the marvin configuration, I did not realize I could use a relative path for that, so I will test.